boot/cypress/MCUBootApp/keys.c

```diff
@@ -0,0 +1,170 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+ /*******************************************************************************
+* \copyright
+*
+* (c) 2018, Cypress Semiconductor Corporation
```

Comment:
I'm not sure if this license is compatible with the Apache-2.0 license above this, but Apache can be tricky to apply. This particular license does restrict the content of this file to use on Cypress devices, so we would need to get general agreement by all of the users of MCUboot to be able to put this in the main repo. Given that the file doesn't appear to have any copyrightable content, perhaps the Cypress-specific license could be removed?